src/node_type.rs

```diff
@@ -536,9 +613,13 @@ impl InternalNode {
     }
 
     /// Gets the child without its corresponding siblings (like using
-    /// [`get_child_with_siblings`](InternalNode::get_child_with_siblings) and dropping the
+    /// [`get_only_child_with_siblings`](InternalNode::get_only_child_with_siblings) and dropping the
     /// siblings, but more efficient).
-    pub fn get_child_without_siblings(&self, node_key: &NodeKey, n: Nibble) -> Option<NodeKey> {
+    pub fn get_only_child_without_siblings(
```

Comment:
So this one is a bit tricky, but basically there is two different type of `get_child` functions:
- The get_child* functions which basically returns the element pointed by n. If there is no child at that exact nibble number it will return None
- The get_only_child* functions which does the same as the previous one, but will return the closest child to the spot pointed by n if this spot is empty (needed for non-existence proofs)